aten/src/ATen/templates/BackendSelectRegister.cpp

```diff
@@ -0,0 +1,22 @@
+// This is used for factory methods dispatching. We register ops with a high priority dispatch key,
```

Comment:
can you add `// ${generated_comment}` and make sure it gets filled in correctly -- look at other examples.